app/components/common/paperItem/index.tsx

```diff
@@ -88,12 +92,14 @@ class RawPaperItem extends React.PureComponent<PaperItemProps> {
         <PaperActionButtons
           currentUser={currentUser}
           paper={paper}
+          hasCollection={!!hasCollection}
```

Comment:
!! Is not it going back to the origin again?

Reply:
It forcingly makes a data type to boolean. 

(ex: `null` is not `boolean`. but `!!null` is boolean.